### PR TITLE
Add unittest :  umount  /

### PR DESCRIPTION
**Pull request by meganeshkadam**
This is the unit test to see if user can unmount `/` partition on fedora linux.

### CODE REVIEW

File: testumountroot.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
```
Comment (kushaldas) [inline testumountroot.py]: Remove this, it will be part of the testsuite, so no need for this.

**Comment by kushaldas** [comment]
Can you please explain the steps of this test here as a comment?

**Comment by meganeshkadam** [comment]
Hi Kushal,

In the first patch, I was trying to compare the same command in out and out1 using assertEqual. This was with reference to snarwade's code, in which he was checking the output for two commands. But later i realized, in this case there is no need to use out1. I can check using out itself.

In second patch, I have removed the out1, and used assertFalse, as the out returns empty string because it is False. out cannot return bool type since the method system returns a tuple of strings only.
it using out itself.

File: testumountroot.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+class TestUmountRoot(unittest.TestCase):
+
+    """
+    This is the test_umountroot class
+
+    """
+
+    def test_umountroot(self):
+        out,retcode,err = system("umount /")
```
Comment (kushaldas) [inline testumountroot.py]: The left hand side of the = sign is clearly wrong.

File: testumountroot.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        out = out.decode('utf-8')
+        self.assertFalse(out)
```
Comment (kushaldas) [inline testumountroot.py]: Instead of the output text, please test of the returncode, it should be 0

**Comment by meganeshkadam** [comment]
@kushaldas  I have modified the code to check retcode should not be equal to 0 as suggested by you. Please check.

**Comment by trishnaguha** [comment]
LGTM :+1: @kushaldas We can merge this.

**Comment by kushaldas** [comment]
@trishnaguha go ahead and merge :)

**Pull request merged**